src/runtime/types/slideover.d.ts

```diff
@@ -10,7 +10,10 @@ export interface Slideover {
   modelValue?: boolean
 }
 
-export interface SlideoverState {
+export interface SlideoverInstance {
+  id: number,
+  isOpen: boolean
   component: Component | string
   props: Slideover
-}
+  close: () => void,
```

Comment:
if `patch` is added here, how would I type it to be able to get type inference for the proper component context?

Reply:
In the current implementation we don't actually export the `createInstance` method. Its used internally when calling `instance.open(...)` and an instance is returned from `.open()`.

Reply:
Yes that's why you should type at least `open`, typing `createInstance` will just make it "match"